src/coreclr/debug/di/process.cpp

```diff
@@ -11377,7 +11414,15 @@ HRESULT CordbProcess::Filter(
 
             // holder will invoke DeleteIPCEventHelper(pManagedEvent).
         }
+#if defined(TARGET_WINDOWS) && defined(TARGET_AMD64)
+        else if (dwFirstChance && pRecord->ExceptionCode == STATUS_BREAKPOINT && pRecord->ExceptionAddress == m_runtimeOffsets.m_setThreadContextNeededAddr) /*|| pRecord->ExceptionCode == STATUS_SINGLE_STEP*/
```

Comment:
Can you remove the commented out code: `/*|| pRecord->ExceptionCode == STATUS_SINGLE_STEP*/`